code/modules/antagonists/pirate/pirate_roles.dm

```diff
@@ -107,9 +107,9 @@
 	outfit = /datum/outfit/pirate/interdyne
 	rank = "Pharmacist"
 
-/obj/effect/mob_spawn/ghost_role/human/pirate/senior
+/obj/effect/mob_spawn/ghost_role/human/pirate/Interdyne/senior
```

Comment:
don't capitalize typepaths

Reply:
what exactly happen when you do?

Reply:
nothing, you just *shouldn't*, I'm 95% sure it's in the style guide not to

Reply:
good enough of a reason uniformity is good